Libs/Visualization/VTK/Core/vtkHistogramMarker.h

```diff
@@ -18,22 +18,30 @@
 
 =========================================================================*/
 
-#ifndef __ctkVTKScalarsToColorsUtils_h
-#define __ctkVTKScalarsToColorsUtils_h
+#ifndef __vtkHistogramMarker_h
+#define __vtkHistogramMarker_h
 
-// CTK includes
-#include "ctkVisualizationVTKWidgetsExport.h"
+#include <vtkObject.h>
+#include <vtkSmartPointer.h>
 
-// VTK includes
-class vtkScalarsToColors;
+class vtkTable;
 
-namespace ctk {
-///
-/// \ingroup Visualization_VTK_Widgets
-/// Convert a vtkScalarsToColors into a QImage
-/// If size is empty, it will use the large icon size of the application style
-QImage CTK_VISUALIZATION_VTK_WIDGETS_EXPORT scalarsToColorsImage(vtkScalarsToColors* scalarsToColors, const QSize& size = QSize());
+class vtkHistogramMarker: public vtkObject
+{
+public:
 
-}
+  static vtkHistogramMarker* New();
 
-#endif
+  vtkHistogramMarker();
+  virtual ~vtkHistogramMarker();
+
+  void SetPosition(double pos);
+
+  vtkSmartPointer<vtkTable> getTable();
+
+private:
+  vtkSmartPointer<vtkTable> m_table;
+
+};
+
+#endif /* __vtkHistogramMarker_h */
```

Comment:
Move this class in vtkDiscretizableColorTransferControlPointsItem.cxx

Reply:
vtkHistogramMarker needed by vtkDiscretizableColorTransferChart. It should be defined in vtkDiscretizableColorTransferChart.cpp or stay in its own file.